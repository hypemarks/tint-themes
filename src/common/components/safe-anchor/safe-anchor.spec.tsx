import { render } from '@testing-library/react';
import React from 'react';
import SafeAnchor from './safe-anchor';

describe('Test <SafeAnchor /> Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(<SafeAnchor className={'className'} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
