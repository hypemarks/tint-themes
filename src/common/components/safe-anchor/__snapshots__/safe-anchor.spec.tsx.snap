// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SafeAnchor /> Component should match snapshot 1`] = `
<DocumentFragment>
  <a
    class="className"
    href="#"
    role="button"
    tabindex="0"
  />
</DocumentFragment>
`;
